src/test/java/study/StringCalculatorTest.java

```diff
@@ -0,0 +1,49 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class StringCalculatorTest {
+    private StringCalculator stringCalculator = new StringCalculator();
+    private final int ZERO = 0;
+
+    @Test
+    void 빈문자열_또는_NULL입력받았을때_0으로반환한다() {
+        assertThat(stringCalculator.calculate(null)).isEqualTo(ZERO);
+        assertThat(stringCalculator.calculate("")).isEqualTo(ZERO);
+    }
```

Comment:
0을 검증할 땐 isZero()를 활용해 보시는 것도 추천드립니다 😃 
```suggestion
    @Test
    void 빈문자열_또는_NULL입력받았을때_0으로반환한다() {
        assertThat(stringCalculator.calculate(null)).isZero();
        assertThat(stringCalculator.calculate("")).isZero();
    }
```